src/js/constants/selectors-map.ts

```diff
@@ -107,6 +107,10 @@ export const quantityInput = {
   alert: (param: string): string => `#js-product-line-alert--${param}`,
 };
 
+export const formValidation = {
+  default: '.needs-validation',
```

Comment:
Mmmmh, not a big fan of this naming... nitpick but I would see something like `form-validation`